.github/ISSUE_TEMPLATE/hotshot-general-task-template.md

```diff
@@ -0,0 +1,27 @@
+---
+name: HotShot General Task Template
+about: For general tasks in HotShot
+title: "[MACRO_TASK_SHORT_NAME] Short task description"
+labels: ''
+assignees: elliedavidson
+
+---
+
+### Describe the task
+* What is the reason for this task? 
+* What work will need to be done to complete this task?  If this is unknown, describe what information is needed to determine the work needed
+
+* For macro tasks - the description should be written for someone unfamiliar with the HotShot repository and roadmap.
+* For non-macro tasks - the description should be written for someone unfamiliar with the task but familiar with HotShot
+
+### Acceptance Criteria
+Describe the acceptance criteria needed for this task to be closed
+* List any sub-tasks that need to be completed 
+* List which tests are applicable to this change.  List any new tests that need to be added for this task
+
+### Checklist
+- [ ] Label this issue appropriately.  If this issue is created by an outside contributor, label it `outside-issue`. Unless a task is labeled `low-priority`, `outside-issue`, or `tech-debt` it must include a `sprint` label and be associated with the `sprint` milestone. 
+- [ ] If this is a macro task, add [MACRO] to the front of the issue name
+- [ ] If this is not a macro task, add [<MACRO_TASK_SHORT_NAME>] to the front of the issue name. Then add a link to the macro task it is associated with.  Every non-macro task must be associated with a macro task unless it is labeled `low-priority`,`tech-debt`, or `outside-issue`. 
```

Comment:
How do we feel about a nested subtasks? I don't have a strong opinion here, but it may be a good idea to avoid too many layers of tasks, so if we have something like:
> Macro task A -> subtask B -> subtasks C and D

We could change it to either
> Macro task A -> subtask B -> _bullet points_ C and D

or
> Macro task A -> subtask B, _C, and D_

for easier tracking.

Reply:
Yes, I like all of these suggestions!